lib/models/Cat.js

```diff
@@ -0,0 +1,26 @@
+const pool = require('../utils/pool');
+
+module.exports = class Cat {
+  id;
+  name;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.type = row.type;
+    this.url = row.url;
+    this.year = row.year;
+    this.lives = row.lives;
+    this.isSidekick = row.is_sidekick;
+  }
```

Comment:
Great job using snake_casing for our row.is_sidekick as postgres does not recognize camelCasing.